extensions-core/druid-bloom-filter/src/main/java/org/apache/druid/query/aggregation/bloom/BloomFilterAggregateCombiner.java

```diff
@@ -24,11 +24,12 @@
 import org.apache.druid.segment.ColumnValueSelector;
 
 import javax.annotation.Nullable;
+import java.nio.ByteBuffer;
 
-public class BloomFilterAggregateCombiner extends ObjectAggregateCombiner<BloomKFilter>
```

Comment:
AggregateCombiner is only used at ingestion time AFAIK. Since this aggregator isn't meant to be used at ingestion time, you might as well delete this and make `makeAggregateCombiner` throw an UnsupportedOperationException.

Reply:
I wasn't certain so left it in the original PR. However, all of my tests pass with it removed, as well as test queries against a debugging cluster, so I think you're right and it should likely be ok removed.